modules/nupac/nupac.nu

```diff
@@ -103,7 +85,7 @@ def packages-to-process [
 
 # downloads fresh repository cache
 def update-repo [] {
-    fetch https://raw.githubusercontent.com/skelly37/nupac/main/nupac.nuon
+    fetch https://raw.githubusercontent.com/skelly37/nupac/metadata-to-nuons/repo-cache.nuon
```

Comment:
revert to main before merge

Reply:
Maybe we could use an environment variable here and default it to main if it's not set

Reply:
And where would you declare/keep the variable?